readme.md

```diff
@@ -10,7 +10,7 @@ Based on [`fast-glob`](https://github.com/mrmlnc/fast-glob) but adds a bunch of
 - Multiple patterns
 - Negated patterns: `['foo*', '!foobar']`
 - Expands directories: `foo` → `foo/**/*`
-- Supports `.gitignore`
+- Supports `.gitignore`, or any similar ignore file system.
```

Comment:
This reads like "file system", not "ignore file  .. system". Use a different word than system.

Reply:
Changed to "configuration", it was the best I could think of.